RELEASE_NOTES.rst

```diff
@@ -217,13 +216,14 @@ Bugfixes:
    ``stderr``.
    (https://github.com/coala-analyzer/coala-bears/issues/251)
 
+
 coala-bears 0.1.0 beta
 =======================
 
 coala-bears is a Python package containing all the bears that are used by coala.
 It has been split from `coala <https://github.com/coala-analyzer/coala>`_.
 With the initial release, it features 56 bears covering 32 languages.
-You can see all of them `here <https://gist.github.com/Adrianzatreanu/cf2d0c8b2ecd542a4860>`_
```

Comment:
Actually this change, i was a bit skeptical about.Not necessary as such but having a little OCD of seeing a rstcheck reveal that there were two instances of "here"(hypertext-link) in files,so I changes it.